input/pagecontent/general-requirements.md

```diff
@@ -26,7 +26,7 @@ There are two different ways to implement AU Core:
 
 Systems may deploy, and support, one or more AU Core profiles (i.e. the profiles governed by this guide) to represent digital health information. A system that implements _profile only support_ will use a profile's content model without any expectations to implement AU Core interactions. 
 
-A system **SHOULD** support all AU Core profiles unless the system does not anticipate supplying or consuming a certain type of data, usually by virtue of playing a limited or specialised role in clinical or information workflows. For example, a pathology laboratory may support [AU Core DiagnosticReport](StructureDefinition-au-core-diagnosticreport.html), but may not support [AU Core MedicationRequest](StructureDefinition-au-core-medicationrequest.html).
+A system **SHOULD** support all AU Core profiles unless the system does not anticipate supplying or consuming a certain type of data, usually by virtue of playing a limited or specialised role in clinical or information workflows.
```

Comment:
Replace the example with what is in AU Core Scope. The easy one is to swap AU Core DiagnosticReport for AU Core Pathology Result Observation.

Reply:
This comment has not been actioned.

Reply:
Have a look at: https://github.com/hl7au/au-fhir-core/pull/110/commits/dc9af83b3bc8a27b6aa08689cdc1249ed1694248#diff-4ce5fd30391d058ae7d545d906e7c4523e7772a36bcf84113e5f328b2141c5dfR29